esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -1851,6 +1851,26 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor* udesc, co
             FileDetails.setNodeGroup(cluster);
         else if (clusters.length() == 1)
             FileDetails.setNodeGroup(clusters.item(0));
+        if (version >= 1.34)
```

Comment:
This isn't that much information or overhead.  So you really don't need this check. But it's ok if you prefer to leave it for this.